app/src/main/java/org/apache/roller/weblogger/pojos/WeblogBookmarkFolder.java

```diff
@@ -188,4 +189,9 @@ public List<WeblogBookmark> retrieveBookmarks() throws WebloggerException {
         return bmgr.getBookmarks(this);
     }
 
+    public void sanitize() {
+        // Conditionally sanitize fields not validated by Struts Validator
+        setName(HTMLSanitizer.conditionallySanitize(getName()));
+    }
```

Comment:
question: can't this be done in the setters? This would ensure that the fields are always clean without having to call the method. Or is this due to serialization or something like that?

Reply:
I think you're right and that is a better approach. I will adjust.